.github/workflows/MainDistributionPipeline.yml

```
@@ -0,0 +1,22 @@
+#
+# This workflow calls the main distribution pipeline from DuckDB to build, test and (optionally) release the extension
+#
+name: Main Extension Distribution Pipeline
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref || '' }}-${{ github.base_ref || '' }}-${{ github.ref != 'refs/heads/main' || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  duckdb-main-pipeline:
+    name: DuckDB Main Pipeline
+    uses: samansmink/duckdb/.github/workflows/_extension_distribution.yml@extension-workflows-2
+    with:
+      duckdb_version: 6855f749756c073bbb5a7abf4109580d45465c45
```

```suggestion
      duckdb_version: 6855f74975
```
Maybe 10 are enough, like the result returned by `PRAGMA version`?

Plus minor, but unsure why this is ducdkb_version ad below extension_ref.

---

yea good point, 10 should be enough here. Not that this should be updated to v0.9.0 as soon as it is released.

Ill update it once the companion PR is merged with the short version of the merge commit on main branch of duckdb/duckdb